### ISSUE TITLE
Daily meeting (9/12/2016)

### ISSUE THREAD

**Issue by Lai0723** [Lai0723/Online-Art-Gallery-Shopping#9]
So every thing go well?

**Comment by pqyyieeuyi** [comment]
I finished the validation part for the user payment yesterday and i just sent to you all , can test it and tell me if got any problem.

**Comment by boonsenglim** [comment]
I just tested ben payment validation and work just fine.

**Comment by pqyyieeuyi** [comment]
Today i working with kc to display the payment history.

**Comment by kcchan3012** [comment]
Today i working with kc to display the payment history.

**Comment by pqyyieeuyi** [comment]
Just now when succesfully insert the payment data to the database and we working to retrive the data from the database.

**Comment by Lai0723** [comment]
Well progress for you two.

**Comment by boonsenglim** [comment]
Well with the help of wei chun, I able to display the gallery in the cart and now i trying writing code to remove the gallery from the cart when the user press the remove button.

**Comment by pqyyieeuyi** [comment]
There is one problem, if we need to retrive the data we need use sql code or got other ways?

**Comment by boonsenglim** [comment]
Ya, you need to by using the gallery id.

**Comment by Lai0723** [comment]
I disagree with boon seng, actually you should use payment id that store on the payment table.

**Comment by kcchan3012** [comment]
Ya, i agree with wei chun too because using gallery id only can know the information that store in gallery table only.

**Comment by boonsenglim** [comment]
I think you two are right , i had misunderstood the concept.

**Comment by pqyyieeuyi** [comment]
I think thats all for the problem now, if got any problem i will contact you all.

**Comment by boonsenglim** [comment]
I also no problem so far.

**Comment by Lai0723** [comment]
So i think that all for today , everyone can dismiss now.
